dhis-2/dhis-support/dhis-support-field-filtering/src/main/java/org/hisp/dhis/fieldfiltering/FieldFilterMixin.java

```diff
@@ -25,14 +25,14 @@
  * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
  * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
-package org.hisp.dhis.commons.jsonfiltering.model;
+package org.hisp.dhis.fieldfiltering;
 
-import lombok.AllArgsConstructor;
-import lombok.Data;
+import com.fasterxml.jackson.annotation.JsonFilter;
 
-@Data
-@AllArgsConstructor
-public class Inner
+/**
+ * @author Morten Olav Hansen
+ */
+@JsonFilter( "field-filter" )
```

Comment:
This deserves some javadoc explaining what and how connections are made based on this annotation being present.